employees/common/exports.py

```diff
@@ -8,217 +6,212 @@
-
-
-def fill_headers(worksheet: Worksheet, headers: list, width_settings: list, employee_name: str):
-    set_employee_name_is_worksheet(worksheet, employee_name)
-    for col_num, column_title in enumerate(headers, 1):
-        cell = worksheet.cell(row=ExcelGeneratorSettingsConstants.HEADERS_ROW.value, column=col_num)
-        cell.value = column_title
-        set_format_styles_for_main_cells(cell, is_header=True)
-        set_columns_width(worksheet, col_num, width_settings)
-
-
 def set_and_fill_description_cell(cell: Cell, cell_value: str):
-    wrapped_alignment = Alignment(vertical=ExcelGeneratorSettingsConstants.VERCTICAL_TOP.value, wrap_text=True)
+    wrapped_alignment = Alignment(vertical=constants.VERCTICAL_TOP.value, wrap_text=True)
     cell.alignment = wrapped_alignment
     cell.value = cell_value
 
 
 def set_and_fill_hours_cell(cell: Cell, cell_value: str):
     if cell_value is not None:
-        cell.value = ExcelGeneratorSettingsConstants.TIMEVALUE_FORMULA.value.format(cell_value)
+        cell.value = constants.TIMEVALUE_FORMULA.value.format(cell_value)
     else:
         cell.value = cell_value
-    cell.number_format = ExcelGeneratorSettingsConstants.HOURS_FORMAT.value
-
-
-def fill_current_report_data(storage_data: dict):
-    for col_num, cell_value in enumerate(storage_data["data"], 1):
-        cell = storage_data["worksheet"].cell(row=storage_data["current_row"], column=col_num)
-        if col_num == storage_data["description_column"]:
-            set_and_fill_description_cell(cell, cell_value)
-        elif col_num == storage_data["hours_column"]:
-            set_and_fill_hours_cell(cell, cell_value)
-        elif col_num == storage_data["daily_hours_column"]:
-            set_and_fill_hours_cell(cell, cell_value)
-        else:
-            cell.value = cell_value
-
-
-def summarizing_reports(worksheet: Worksheet, last_row: int, hours_column: int, formula: str):
-    total_cell = worksheet.cell(row=last_row, column=ExcelGeneratorSettingsConstants.TOTAL_COLUMN.value)
-    total_cell.value = ExcelGeneratorSettingsConstants.TOTAL.value
-    for column_number in range(1, worksheet.max_column + 1):
-        set_format_styles_for_main_cells(worksheet.cell(row=last_row, column=column_number), is_header=False)
-
-    total_hours_cell = worksheet.cell(row=last_row, column=hours_column)
-    total_hours_cell.value = formula.format(last_row - 1)
-    total_hours_cell.number_format = ExcelGeneratorSettingsConstants.TOTAL_HOURS_FORMAT.value
-    set_format_styles_for_main_cells(total_hours_cell, is_header=False)
+    cell.number_format = constants.HOURS_FORMAT.value
 
 
 def get_employee_name(author: CustomUser) -> str:
-    if author.last_name is not None and author.first_name is not None:
-        return f"{author.first_name} {author.last_name}"
+    if author.last_name and author.first_name:
+        return f"{author.first_name} {author.last_name[0]}."
```

Comment:
Why we have to get first element from `last_name`?

Reply:
Wiem, że w adresie e-mailowym mamy również nazwisko, ale Adrian powiedział, że powinniśmy wyświetlać go w ten sposób ponieważ mamy RODO. :P

Reply:
I know that we also have a name in the e-mail address, but Adrian said we should display it this way because we have the RODO.

Reply:
It's the first letter of the surname